evm-contracts/src/v0.7/dev/Operator.sol

```diff
@@ -402,6 +403,15 @@ contract Operator is
     ));
   }
 
+  /**
+   * @notice Returns the LINK available in this contract, not locked in escrow
+   * @return uint256 LINK tokens available
+   */
+  function fundsAvailable() internal view returns (uint256) {
```

Comment:
```suggestion
  function fundsAvailable() private view returns (uint256) {
```
I think we should default to private by default and only make things internal or public when we need to.